projects/common/src/utilities/formatters/enum/display-string-enum.pipe.ts

```diff
@@ -5,7 +5,7 @@ import { displayStringEnum } from './display-string-enum';
   name: 'htDisplayStringEnum'
 })
 export class DisplayStringEnumPipe implements PipeTransform {
-  public transform(value: string): string {
-    return displayStringEnum(value);
+  public transform(value?: string, unknown: string = '-', separator: string = ' '): string {
```

Comment:
variable naming: `unknown` shouldn't be used as it's a type - is that the `defaulttValue` or similar?

Reply:
Fixed.